UnitWorld_client_gui/graphics/shared/SharedDisplay.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+namespace uw
+{
+	class SharedDisplay
+	{
+	public:
+		virtual void tryClose() = 0;
```

Comment:
What's the link between the `SharedDisplay` class name and tryClose function? What,s this class?